example/lib/pages/full_color_icons_page.dart

```diff
@@ -182,6 +182,7 @@ const iconDirsToIconNames = <String, Set<String>>{
     'accessories-dictionary',
     'accessories-text-editor',
     'address-book-app',
+    'app-center',
```

Comment:
So we have a separate one called app-center? Since there is the snap-store icon too?

Reply:
I just listed all app icons from yaru. Ideal would be some http call trying to list all those icons on its own (github dart api?) but I didnt manage this so it is a static list of all icons from yaru.